cppcache/src/SerializationRegistry.cpp

```diff
@@ -374,118 +368,121 @@ std::shared_ptr<Serializable> SerializationRegistry::GetEnum(
 }
 
 void TheTypeMap::clear() {
-  std::lock_guard<util::concurrent::spinlock_mutex> guard(m_mapLock);
-  m_map->unbind_all();
+  std::lock_guard<util::concurrent::spinlock_mutex> guard(
+      m_UserDataSerializablesMapLock);
+  m_UserDataSerializablesMap->unbind_all();
 
-  std::lock_guard<util::concurrent::spinlock_mutex> guard2(m_map2Lock);
-  m_map2->unbind_all();
+  std::lock_guard<util::concurrent::spinlock_mutex> guard2(m_InternalsMapLock);
+  m_InternalsMap->unbind_all();
 
   std::lock_guard<util::concurrent::spinlock_mutex> guard3(m_pdxTypemapLock);
   m_pdxTypemap->unbind_all();
 }
 
-void TheTypeMap::find(int64_t id, TypeFactoryMethod& func) const {
-  std::lock_guard<util::concurrent::spinlock_mutex> guard(m_mapLock);
-  m_map->find(id, func);
+void TheTypeMap::findSerializable(int32_t id, TypeFactoryMethod& func) const {
+  std::lock_guard<util::concurrent::spinlock_mutex> guard(
+      m_UserDataSerializablesMapLock);
+  m_UserDataSerializablesMap->find(id, func);
 }
 
-void TheTypeMap::find2(int64_t id, TypeFactoryMethod& func) const {
-  std::lock_guard<util::concurrent::spinlock_mutex> guard(m_map2Lock);
-  m_map2->find(id, func);
+void TheTypeMap::findInternal(int32_t id, TypeFactoryMethod& func) const {
```

Comment:
Similar theme of using names that reflect `DataSerializableFixedId` and `DataSerializable`.